_stbt/match.py

```diff
@@ -544,14 +540,18 @@ def imwrite(name, img, scale=1):
             imglog.imwrite("level%d-%s" % (level, name), img, scale=scale)  # pylint:disable=cell-var-from-loop
 
         heatmap, heatmap_scale = _match_template(
-            image_pyramid[level], template_pyramid[level], mask, method,
-            roi_mask, level, imwrite)
+            image_pyramid[level], template_pyramid[level], mask_pyramid[level],
+            method, roi_mask, level, imwrite)
 
         # Relax the threshold slightly for scaled-down pyramid levels to
         # compensate for scaling artifacts.
-        threshold = max(
-            0,
-            match_parameters.match_threshold - (0.2 if level > 0 else 0))
+        if level == 0:
+            relax = 0
+        elif match_parameters.match_method == MatchMethod.SQDIFF:
+            relax = 0.01
```

Comment:
How did you choose this relax value?  Strictly this isn't a change for transparency, it's general performance with SQDIFF, but it doesn't need to be split out.

Reply:
It seems to work. I think that, given we're starting "from scratch" with a new match method, we want to avoid repeating the mistake of making it too permissive. This has significant performance benefits because the pyramid optimisation works more often. Ultimately I'd like to even remove the second pass (perhaps only enable it when we detect that the reference image has very low contrast -- see [this comment](https://github.com/stb-tester/stb-tester/pull/544#issuecomment-447799743) on #544).

Before we release v30 I'll do more dogfooding and we can adjust if necessary.